src/api/departments.ts

```diff
@@ -1,53 +1,39 @@
 import express from "express";
-import sql, { one } from "../database.js";
+import sql from "../database.js";
 
 const router = express.Router();
 export default router;
 
 interface Department {
   id: number;
   name: string;
-  college_prefix: string;
 }
 
 interface Program {
   id: number;
+  department_id: number;
   name: string;
   type: string;
 }
 
 router.get("/departments", async (req, res) => {
-  const departments = await sql<Department[]>`SELECT id, name, college_prefix FROM departments`;
+  const departments = await sql<Department[]>`
+    SELECT id, name FROM departments`;
 
   res.send(
     departments.map((department) => ({
       id: department.id,
       name: department.name,
-      college: department.college_prefix,
     })),
   );
 });
 
-router.get("/departments/:departmentId", async (req, res) => {
-  const departmentId = req.params.departmentId;
-
-  const department = await one<Omit<Department, "id">>(
-    sql`SELECT name, college_prefix FROM departments WHERE id = ${departmentId}`,
-  );
-
-  res.send({
-    name: department.name,
-    college: department.college_prefix,
-  });
-});
-
 router.get("/departments/:departmentId/programs", async (req, res) => {
   const departmentId = req.params.departmentId;
 
-  const programs = await sql<Program[]>`
+  const programs = await sql<Omit<Program[], "department_id">>`
```

Comment:
Why are we adding a new field just to omit it here?

Reply:
I believe our interfaces should reflect our database, and we can use Pick and Omit when needed.

Reply:
Not a bad idea actually, before I was using interfaces for the result of each individual query.
If we do that though, we should pull the database type definitions out into their own file/files.

Reply:
I like that idea, could you open an issue for that?